### ISSUE TITLE
Can't change profile 

### ISSUE THREAD

**Issue by soutiago33** [Mobile-Artificial-Intelligence/maid#596]
can't change it successfully, the pic also doesn't change or save

**Comment by danemadsen** [comment]
> can't change it successfully, the pic also doesn't change or save

Yeah this was a bug in 1.2.9 its fixed in the latest builds and 1.3.0 when it releases
